### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by Ishiihara**
@ewencp @gwenshap Doc improvements including schema evolution and Hive integration.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -6,16 +6,27 @@ HDFS Connector
...
 In this Quickstart, we use the HDFS connector to export data produced by the Avro console producer
 to HDFS.
 
 Start Zookeeper, Kafka and SchemaRegistry is you haven't done so. The instructions on how to start
-these services is available in this document. You also need to have Hadoop running locally or
-remotely and makes sure that you know the HDFS url. For Hive integration, you also need to have
-Hive installed. This Quickstart assumes that you started the required services with the default
-config and you should make necessary changes according to the actual configuration used.
+these services is available at the Confluent Platform QuickStart. You also need to have Hadoop
+running locally or remotely and makes sure that you know the HDFS url. For Hive integration, you
```
Comment (gwenshap) [inline docs/index.rst]: s/makes/make/

File: docs/index.rst [hunk 2]
```diff
@@ -6,16 +6,27 @@ HDFS Connector
...
+need to have Hive installed. This Quickstart assumes that you started the required services with
```
Comment (gwenshap) [inline docs/index.rst]: and to know the metastore URI.

File: docs/index.rst [hunk 3]
```diff
@@ -35,18 +46,43 @@ Then run the following command to start Kafka connect with the HDFS connector::
   $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties \
   etc/kafka-connect-hdfs/quickstart-hdfs.properties
 
-You should see the process start up and log some messages, and then it will export data from
-Kafka to HDFS. In order to check that the data have been copied to HDFS, you can use Hive to query
-data or Avro tools to extract the file in HDFS::
+You should see that the process starts up and logs some messages, and then it will export data from
```
Comment (gwenshap) [inline docs/index.rst]: Perhaps before asking them to run the connector, point them to the configuration file and ask them to validate important properties (such as the HDFS URI)?

File: docs/index.rst (hunk 1 above)
Comment (gwenshap) [inline docs/index.rst]: They also need to create /topics and /logs directories and make sure whatever user is running the connector can write to those directories.

File: docs/index.rst [hunk 4]
```diff
@@ -35,18 +46,43 @@ Then run the following command to start Kafka connect with the HDFS connector::
...
+      hive.integration=true
+      hive.metastore.uris=uri to your hive metastore
+      schema.compatibility=BACKWARD
+   After the connector finishes ingesting data to HDFS, you can use Hive to check the data::
 
-You can use ``avro-tools-1.7.7.jar`` to extract the content of the file::
+      $hive>SELECT * FROM test_hdfs;
 
-  $ hadoop jar avro-tools-1.7.7.jar tojson /topics/test_hdfs/partition=0/test_hdfs+0+0000000000+0000000002.avro
+   Note that if you leave ``hive.metastore.uris`` empty, an embedded Hive metastore will be created
```
Comment (gwenshap) [inline docs/index.rst]: ???
What creates the metastore? our connector?

Comment (Ishiihara) [inline docs/index.rst]: Yes. If the hive metastore uri is missing, the connector uses derby as the embedded metastore.

File: docs/index.rst (hunk 1 above)
Comment (Ishiihara) [inline docs/index.rst]: The connector creates ``topics.dir`` and ``logs.dir`` for you.

File: docs/index.rst (hunk 4 above)
Comment (gwenshap) [inline docs/index.rst]: This will become a support issue... lets file a bug to fix this behavior for next release.

Comment (Ishiihara) [inline docs/index.rst]: Sure. Filing the JIRA now.

File: docs/index.rst (hunk 1 above)
Comment (gwenshap) [inline docs/index.rst]: only if the connector has privileges to do it. In many Hadoop clusters, it will not have write access to "/"...

**Comment by Ishiihara** [comment]
@gwenshap Updated PR according to review comments. Also, polished a bit on the schema evolution section.

**Comment by gwenshap** [comment]
LGTM. Squashing, merging and committing to both master and 2.x

**Comment by gwenshap** [comment]
ok, merging and pushing is done. But due to the way I did it (merge -squash locally and push), I think you'll need to close the PR and JIRA manually.

**Comment by gwenshap** [comment]
never mind, I can close it :)

**Pull request closed without merging**